script.js

```diff
@@ -53,6 +53,7 @@ const options = {
     shadowblur: {letter: "𐤎", description: "size of blurry shadow applied to line segments (non-zero values might not play well with some blend modes) <i>in pixels</i>", min: 0, max: 50, step: 0.1, default: 0},
     linecap: {letter: "𐤂", description: "line cap (1: butt, 2: round, 3: squre)", min: 1, max: 3, step: 1, default: 1},
     fadeinspeed: {letter: "𐤁", description: "rate at which line segments randomly appear <i>in iterations</i>", min: 0, max: 200, step: 1, default: 0},
+    hueshiftspeed: {letter:"ܘ", description: "hue shift speed <i>in deegrees per iteration</i>", min: -10, max: 10, step: 0.1, default: 0},
```

Comment:
I'm only loading webfonts for [Imperial Aramaic](https://en.wikipedia.org/wiki/Imperial_Aramaic#Unicode) and [Phoenician](https://en.wikipedia.org/wiki/Phoenician_alphabet#Unicode) – the symbol you chose appears to be part of *Syriac Aramaic*, which isn't really a problem if it weren't for the fact that it might not render properly on some platforms. Could you choose a different unused symbol from the two scripts I mentioned? (I can also do this myself later on.)

Reply:
Thanks, I wasn't sure where the symbols came from. I changed it to 𐡌 from [Imperial Aramaic](https://en.wikipedia.org/wiki/Imperial_Aramaic#Unicode). Of course feel free to change it as you wish.